DaysOfCode/operators.js

```diff
@@ -7,35 +7,35 @@ let inputString = '';
 let currentLine = 0;
 
 process.stdin.on('data', (inputStdin) => {
-    inputString += inputStdin;
+  inputString += inputStdin;
 });
 
-process.stdin.on('end', _ => {
-    inputString = inputString.replace(/\s*$/, '')
-        .split('\n')
-        .map((str) => str.replace(/\s*$/, ''));
+process.stdin.on('end', () => {
+  inputString = inputString.replace(/\s*$/, '')
+    .split('\n')
+    .map((str) => str.replace(/\s*$/, ''));
 
-    main();
+  main();
 });
 
 function readLine() {
-    return inputString[currentLine += 1];
+  return inputString[currentLine += 1];
 }
 
 // Complete the solve function below.
 function solve(meal_cost, tip_percent, tax_percent) {
-    let meal = parseFloat(meal_cost).toFixed(2);
-    let totTip = (meal * (parseFloat(tip_percent) / 100)).toFixed(2);
-    let totTax = (meal * (parseFloat(tax_percent) / 100)).toFixed(2);
-    let resul = Math.round(parseFloat(meal) + parseFloat(totTip) + parseFloat(totTax));
+  let meal = parseFloat(meal_cost).toFixed(2);
+  let totTip = (meal * (parseFloat(tip_percent) / 100)).toFixed(2);
+  let totTax = (meal * (parseFloat(tax_percent) / 100)).toFixed(2);
+  let resul = Math.round(parseFloat(meal) + parseFloat(totTip) + parseFloat(totTax));
```

Comment:
why are you converting the totTip to string and then parsing it to a float later?